package.json

```diff
@@ -21,11 +21,14 @@
     "type-check": "tsc --noEmit",
     "update-sitemap": "ts-node scripts/updateSitemap.ts",
     "watch": "tsc --watch",
-    "update-sailthru": "ts-node scripts/pushCollectionsToSailthru.ts"
+    "update-sailthru": "ts-node scripts/pushCollectionsToSailthru.ts",
```

Comment:
Definitely out of scope of this PR, but does this noise point to a difference in linting rules or prettier settings for KAWS as compared to our other JavaScript projects?

Reply:
oh no it just added the comma because a new script was added to the end, right?